xapian-glib/xapian-utils.cc

```diff
@@ -131,3 +131,33 @@ xapian_revision (void)
 {
   return Xapian::revision ();
 }
+
+/**
+ * xapian_sortable_serialize:
+ *
+ * Convert a floating point number to a string, preserving sort order.
+ *
+ * This method converts a floating point number to a string, suitable for using
+ * as a value for numeric range restriction, or for use as a sort key.
+ */
```

Comment:
Missing `Returns:` and `Since:` annotations:

    * Returns: (transfer full): the serialized value
    *
    * Since: 1.4